Makefile

```diff
@@ -0,0 +1,58 @@
+.DEFAULT_GOAL := check
+
+# tools
+curl ?= curl
+xpath ?= xpath
+cddl ?= cddl
+diag2cbor ?= diag2cbor.rb
+
+# upstream repo configuration
+eat_repo_baseurl ?= https://ietf-rats-wg.github.io/eat
+# Set eat_repo_branch if you need to work with a branch other than master/main.
+# For example, if you need to work with a branch named CoSWID, do:
+#   make eat_repo_branch=CoSWID
+# Check https://ietf-rats-wg.github.io/eat/ for the branches that are currently
+# available.
+eat_repo_branch ?=
+ifeq ($(eat_repo_branch),)
+  eat_repo := $(eat_repo_baseurl)
+else
+  eat_repo := $(eat_repo_baseurl)/$(eat_repo_branch)
+endif
+
+eat_xml := draft-ietf-rats-eat.xml
+CLEANFILES += $(eat_xml)
+
+eat_cddl := eat.cddl
+CLEANFILES += $(eat_cddl)
+
+$(eat_cddl): $(eat_xml)
+	$(xpath) -n -q -e '//section[@anchor="collected-cddl"]//artwork/text()' $< \
+		| sed -e 's/\&amp;/\&/g' \
+		> $@
```

Comment:
I get this error which seems to say my xpath doesn't know about rfc2629 type of documents:

xpath draft-ietf-rats-eat.xml -n -e -q '//section[@anchor="collected-cddl"]//artwork/text()' \
		| sed -e 's/\&amp;/\&/g' \
		> eat.cddl

404 File `/Users/lgl/Code/ET/makefile/rfc2629-xhtml.ent' does not exist file:///Users/lgl/Code/ET/makefile/rfc2629-xhtml.ent
Handler couldn't resolve external entity at line 4, column 40, byte 212
error in processing external entity reference at line 4, column 40, byte 212:
<?xml-stylesheet type="text/xsl" href="rfc2629.xslt" ?>
<!-- generated by https://github.com/cabo/kramdown-rfc2629 version 1.4.1 -->
<!DOCTYPE rfc SYSTEM "rfc2629-xhtml.ent">
=======================================^
<?rfc toc="yes"?>
<?rfc tocdepth="4"?>
 at /System/Library/Perl/Extras/5.18/darwin-thread-multi-2level/XML/Parser.pm line 187.

Reply:
We must run different `xpath` versions.  The `-n` flag in mine instructs xpath to ignore any external DTD dependency.

Reply:
I have added a docker sandbox in 1b0d9a012 to have a shared dev environment